### PR TITLE
Content and style updates

### PR DESCRIPTION
**Pull request by spencercanner**
#### Changes:
* updates the home page content to be consistent with our mission
* updates screenshot on home page
* makes mobile site text more legible
* more mobile specific improvements to the circle display 

#### Some screenshots from mobile:
img_5484
img_5485
img_5487
img_5486
img_5489
img_5491
img_5490
img_5492
img_5494
img_5493
img_5495

---
#### 🔥 tip if you want to check this out on your phone: 
* start the app using `flask run --host=0.0.0.0`
* go to `yourComputerAddress:5000`(i.e. `spencer-imac.local:5000`) on your phone
* yourComputerAddress can be found in System Preferences > Sharing

### CODE REVIEW

File: static/script/bubbles.js [hunk 1]
```diff
@@ -40,7 +40,8 @@ function createCirclePacking(root) {
   // 850
   var windowWidth = Math.max(document.documentElement.clientWidth, window.innerWidth || 0);
   var width = windowWidth > 850 ? 850 : windowWidth;
-  var magicNumber = windowWidth > 850 ? 45 : 20;
+  const isSmall = width == 850 ? false : true;
```
Comment (oghosa) [inline static/script/bubbles.js]: oooh was this what fixed the text on? @spencercanner

File: templates/index.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
 {% endblock %}
 
 {% block header_title %}Welcome to newvella{% endblock %}
-{% block header_subtitle %}Let us fill your bookshelf{% endblock %}
```
Comment (oghosa) [inline templates/index.html]: I'll miss you beautiful text.

File: static/script/bubbles.js (hunk 1 above)
Comment (spencercanner) [reply static/script/bubbles.js]: yeah, this allowed me to make mobile specific customizations to the text size and whether the text would appear

File: templates/index.html (hunk 2 above)
Comment (spencercanner) [reply templates/index.html]: I think the new text is a better value proposition especially since it's the first thing users see on the website

**Pull request merged**